docroot/themes/humsci/humsci_basic/src/scss/_main.scss

```diff
@@ -1,4 +1,90 @@
+@charset "UTF-8";
+
 @import
-  "variables.general",
-  "tools.mixins.themes",
-  "base";
+  // DECANTER (matching how they bring in their imports in decanter.scss)
+  //
+  // Normalize, Vendors
+  'decanter/core/src/scss/core/normalize',
+
+  // Utilities
+  // Decanter no markup will not compile any CSS unless used.
+  // It brings in everything in the utilities, like variables, mixins,
+  // functions, and placeholders.
+  'decanter/core/src/scss/decanter-no-markup',
+
+  // Core elements
+  // Decanter font imports and sources
+  'decanter/core/src/scss/core/fonts',
+  'decanter/core/src/scss/core/helpers',
+  'decanter/core/src/scss/core/flex-grid',
+  'decanter/core/src/scss/core/base',
+  'decanter/core/src/scss/elements',
```

Comment:
Per your previous comments:

- `decanter/core/src/scss/core/flex-grid`: Lets leave this out and pull it in later if we need it. It looks like we aren't really using any of these classes in our existing templates.
- `decanter/core/src/scss/core/helpers`: I'm ok with keeping these helpers. There are only a few and they seem fairly useful (for screenreaders, etc).
- `decanter/core/src/scss/elements`: I agree, let's include this for now, but if we find that we're doing tons of overrides, we should consider excluding it. Maybe we add a code comment that communicates this intent?